File: 2894-divisible-and-non-divisible-sums-difference/2894-divisible-and-non-divisible-sums-difference.py
class Solution(object):
    def differenceOfSums(self, n, m):
        sums=0
        num=0
        for i in range(1,n+1):
            if(i%m==0):
                sums=sums+i
            else:
                num=num+i
        total=num-sums
        return total
        

File: 2894-divisible-and-non-divisible-sums-difference/2894-divisible-and-non-divisible-sums-difference.java
class Solution {
    public int differenceOfSums(int n, int m) {
        int dsum=0,ndsum=0;
        for(int i=1;i<=n;i++)
        {
            if(i%m==0)
            {
                dsum+=i;
            }
            else
            {
                ndsum+=i;
            }
        }
        
        int sum=ndsum-dsum;
        return sum;
        

        
    }
}